packages/origin-247-transfer/src/repositories/EnergyTransferRequest.entity.ts

```diff
@@ -5,9 +5,14 @@ import {
     UpdateDateColumn,
     Column
 } from 'typeorm';
-import { EnergyTransferRequestAttrs, TransferValidationStatus } from '../EnergyTransferRequest';
+import {
+    CertificateData,
+    EnergyTransferRequestAttrs,
+    State,
+    TransferValidationStatus
+} from '../EnergyTransferRequest';
 
-export const tableName = 'energy_transfer_request';
+export const tableName = 'energy_transfer_request_v2';
```

Comment:
Noice

Reply:
:) It was too easy :)

I described my goal in PR description